src/sentry/utils/snowflake.py

```diff
@@ -55,3 +94,55 @@ def snowflake_id_generation():
     ID_VALIDATOR.validate(snowflake_id)
 
     return snowflake_id
+
+
+def get_redis_cluster(redis_key: str):
+    return redis.clusters.get("default").get_local_client_for_key(redis_key)
+
+
+def store_snowflake_to_redis(redis_key: str, timestamp: int, used_region_sequence: int) -> None:
```

Comment:
The noun "sequence" seems a little overloaded -- "region sequence" by itself sounds like a collection of values, not just one. Maybe we could change this to "sequence value" or something.